queenbee/recipe/dag.py

```diff
@@ -187,6 +187,18 @@ class DAGTaskArtifactArgument(BaseModel):
         ' want to source an artifact from within that directory.'
     )
 
+    value: str = Field(
```

Comment:
Not sure I understand what the `fixed value` is here. Is it a JSON object? Is it a hardcoded path to something in the project folder?

If it's the latter then I think we should create a `ProjectFolderPathReference` class to put in the `from_` union.